javalib/src/main/scala/java/lang/System.scala

```diff
@@ -30,7 +30,7 @@ object System {
   def getProperty(key: String, default: String): String = ???
   def setProperty(key: String, value: String): String   = ???
 
-  def nanoTime(): CLong = time.scalanative_nano_time
+  def nanoTime(): CLongLong = time.scalanative_nano_time
```

Comment:
This one is Java API, so it should unconditionally return `scala.Long` on all platforms.